modules/initialize_util.py

```diff
@@ -10,7 +10,9 @@
 def gradio_server_name():
     from modules.shared_cmd_options import cmd_opts
 
-    if cmd_opts.server_name:
+    if cmd_opts.ipv6:
+        return "[::]"

```

Comment:
This should not override `server_name`. If anywhere, it should be in the else branch where a global listen address is being computed for `cmd_opts.listen`.

Reply:
> This should not override `server_name`. If anywhere, it should be in the else branch where a global listen address is being computed for `cmd_opts.listen`.

this is same for listen mapping to ipv6 as well